### PR TITLE
WooCommerce: Update paths to keep site at the end, add child items to settings nav

### LINKED ISSUE #14678: Add sub-items & separators to store nav
nav

The sub-items would only be visible when they or their parent item are active.

### LINKED ISSUE #14729: WooCommerce: Update routes to calypso standards
1. Site should always be last
2. Use singular (e.g. order not orders)

cc @ryelle

### PR DESCRIPTION
**Pull request by allendav**
Fixes #14678 
Fixes #14729 

To Test

1. Navigate to /store/:site
2. Make sure each sidebar menu item works (note that everything will display the Dashboard except for Products-Add, Settings-Payments, and Settings-Shipping at this time, but the routes should be correct nonetheless)
3. Make sure each dashboard button works (see caveat in 2)
4. Make sure that the settings child items (Payments, Shipping and Taxes) are only displayed when Settings itself or one of them is selected

Note: Some routes change as a result of this PR:

```
FROM						TO
/store/products/:site			no change
/store/products/:site/add		/store/product/:site (note the singular here)
/store/products/:site/import		/store/products/import/:site

/store/orders/:site				no change
/store/orders/:site/add			/store/order/:site (note the singular here)

/store/settings/:site			no change (this corresponds to general settings)
/store/settings/:site/shipping	/store/settings/shipping/:site
/store/settings/:site/payments	/store/settings/payments/:site
/store/settings/:site/tax			/store/settings/tax/:site
```

### CODE REVIEW

File: client/extensions/woocommerce/index.js [hunk 1]
```diff
@@ -114,17 +103,46 @@ const getStorePages = () => {
 		{
 			container: SettingsPayments,
 			configKey: 'woocommerce/extension-settings-payments',
-			path: '/store/settings/:site/payments',
+			path: '/store/settings/payments/:site',
+			sidebarItem: {
+				isPrimary: false,
+				label: translate( 'Payments' ),
```
Comment (a8ci18n) [inline client/extensions/woocommerce/index.js]: Hi! I've found a possible matching string that has already been translated **43** times:
`translate( 'Payment' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by allendav** [comment]
cc @kellychoffman or @jameskoster for design-fu

**Comment by belcherj** [comment]
I'm good with this.

**Comment by kellychoffman** [comment]
👍 styled and good to go!

**Pull request merged**